### PR TITLE
Gerrit ansible

### PR DESCRIPTION
**Pull request by m-kostrzewa**
Note: review #124 first.

### CODE REVIEW

**Comment by krdln** [comment]
This is based on top of #124.

Note that when switching `gerrit-build-all` to this Jenkinsfile, the branches and repos should be adjusted, as they are now configured as in our CI.

File: CIScripts/Test/TestConfigurationUtils.ps1 [hunk 1]
```diff
@@ -74,7 +74,7 @@ function Enable-VRouterExtension {
            [Parameter(Mandatory = $true)] [string] $ForwardingExtensionName,
            [Parameter(Mandatory = $false)] [string] $ContainerNetworkName = "testnet")
 
-    Write-Output "Enabling Extension"
+    Write-Host "Enabling Extension"
```
Comment (krdln) [inline CIScripts/Test/TestConfigurationUtils.ps1]: If the semantics of `Write-Output` fit us better than `Write-Host`, we can make a separate PR for changing it in the whole repo. Still, I'm not sure which one is more correct there.

**Comment by krdln** [comment]
TODO:

* Capture logs from `build` powershell step
* Rsync logs to the logs server
    * Only if the build was triggered by gerrit
* ~~~Use the same branch for Jenkinsfile and to checkout CIScripts (current approach doesn't work)~~~
* Rebase on master

**Comment by m-kostrzewa** [comment]
moved to: https://github.com/codilime/contrail-windows-tools/pull/134
